sound/hybrid_font.h

```diff
@@ -120,7 +122,37 @@ class HybridFont : public SaberBase {
       PlayPolyphonic(effect);
     }
   }
+  void StartSwing(Effect* monophonic, Effect* polyphonic) override {
+    if (polyphonic->files_found()) {
+      swing_player_ = PlayPolyphonic(polyphonic);
```

Comment:
This can cause some weirdness if swing_player_ is already set.
Not sure what we want to happen in that case exactly.

Reply:
I did this so, when spinning a saber, the sound plays instantly.  Having a check first caused a delay that made swings not play fast enough when spinning.

Reply:
Oh, I'm not saying that we should not start a swing if one is already playing.
The problem is simply that first swing sound will suddenly not have it's volume adjusted anymore.
I'm not sure if that's a problem, or not, but I thought it would be worth mentioning at least.
If we want to fix it, we would need to possibly keep track of more than one playing swing and adjust all their volumes in SetSwingVolume().

Reply:
fixed.  Added a second swing player to handle the case where two swings might play.